rclcpp/include/rclcpp/executor.hpp

```diff
@@ -360,6 +366,9 @@ class Executor
 private:
   RCLCPP_DISABLE_COPY(Executor)
 
+  void
```

Comment:
This needs visibility macros for linking to work properly on Windows:

```
21:45:52 [86.688s] test_component_manager_api.obj : error LNK2019: unresolved external symbol "private: void __cdecl rclcpp::executor::Executor::spin_once_impl(class std::chrono::duration<__int64,struct std::ratio<1,1000000000> >)" (?spin_once_impl@Executor@executor@rclcpp@@AEAAXV?$duration@_JU?$ratio@$00$0DLJKMKAA@@std@@@chrono@std@@@Z) referenced in function "public: enum rclcpp::executor::FutureReturnCode __cdecl rclcpp::executor::Executor::spin_until_future_complete<class std::shared_ptr<struct composition_interfaces::srv::ListNodes_Response_<class std::allocator<void> > >,__int64,struct std::ratio<1,1> >(class std::shared_future<class std::shared_ptr<struct composition_interfaces::srv::ListNodes_Response_<class std::allocator<void> > > > const &,class std::chrono::duration<__int64,struct std::ratio<1,1> >)" (??$spin_until_future_complete@V?$shared_ptr@U?$ListNodes_Response_@V?$allocator@X@std@@@srv@composition_interfaces@@@std@@_JU?$ratio@$00$00@2@@Executor@executor@rclcpp@@QEAA?AW4FutureReturnCode@12@AEBV?$shared_future@V?$shared_ptr@U?$ListNodes_Response_@V?$allocator@X@std@@@srv@composition_interfaces@@@std@@@std@@V?$duration@_JU?$ratio@$00$00@std@@@chrono@5@@Z) [C:\ci\ws\build\rclcpp_components\test_component_manager_api.vcxproj]
21:45:52 [86.688s] C:\ci\ws\build\rclcpp_components\Release\test_component_manager_api.exe : fatal error LNK1120: 1 unresolved externals [C:\ci\ws\build\rclcpp_components\test_component_manager_api.vcxproj]
```

```suggestion
  RCLCPP_PUBLIC
  void
```

Reply:
I applied your comment. Thank you.